packages/terra-clinical-item-view/lib/_mixins.scss

```diff
@@ -0,0 +1,21 @@
+@mixin terra-text-body {
+  font-size: 1.1428571428571428rem;
+  line-height: 1.4285714285714286rem;
+
+  @media screen and (min-width: 760px) {
+    font-size: 1rem;
+  }
+}
+
+@mixin terra-x-small-heading {
```

Comment:
Should we introduce an extra small, or can this be tiny? I'm not sure how we should handle the sizing system for this to be consistent